### PR TITLE
Fix deprecations.

### LINKED ISSUE #218: Warning: Calling puts_columns is deprecated
```bash
Warning: Calling puts_columns is deprecated!
Use puts Formatter.columns instead.
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/commands/cleanup.rb:18:in `run'
Please report this to the homebrew/bundle tap!
```

I'm seeing that when using `brew bundle`.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Fix the deprecations of `puts_columns` and `MultiFormatter[]`.

Fixes #218.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -37,6 +37,11 @@ def require(path)
   end
 end
 
+class Formatter
```
Comment (reitermarkus) [inline spec/spec_helper.rb]: It's a `module`.

Comment (MikeMcQuaid) [inline spec/spec_helper.rb]: Works fine either way; just for stubbing.

Comment (reitermarkus) [inline spec/spec_helper.rb]: Yeah, just wanted it to be consistent with what's in core. Out of curiosity, why exactly is this being stubbed?

Comment (MikeMcQuaid) [inline spec/spec_helper.rb]: The test suite doesn't include Homebrew code.

**Pull request merged**